drive/modules/listener/http_service.bal

```diff
@@ -101,5 +104,17 @@ service class HttpService {
             check caller->respond(http:STATUS_OK);
         }
     }
+    resource isolated function get [string name](http:Caller caller) returns @tainted error? {
+        http:Response r = new();
+        if(self.dnsVerificationFileContent.length() < 100 && 
```

Comment:
`self.dnsVerificationFileContent.length() < 100` what is the reason behind this?

Reply:
In review, it was identified that someone could inject a code in place of `dnsVerificationFileContent ` and get it executed to obtain internal information. We have done required sanitization, this is an additional check. @SkNuwanTissera can explain more.

Reply:
Yes. My intension is to provide some control over the content passed from the front end. Since we are directly returning content a user sets from the front end, This can be lead to XSS vulnerability unless it's sanitized properly. But here, I have restricted with a character limit. So user cannot pass bulky text objects/code snippets to the backend.
By the way, The length we expects here is a fixed length

Reply:
Noted.